### PR TITLE
[WFLY-17102] / [WFLY-17849] Attempt to fix the intermittent failures in the OIDC tests and unignore them

### PR DESCRIPTION
**Pull request by fjuma**
https://issues.redhat.com/browse/WFLY-17102
https://issues.redhat.com/browse/WFLY-17849

Because this PR aims to fix the intermittent failures in the OIDC tests, I'm going to add the hold label to this PR until there are a bunch of successful runs.

### CODE REVIEW

File: testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java [hunk 1]
```diff
@@ -75,15 +77,22 @@ public static RealmRepresentation getRealmRepresentation(final String realmName,
...
-                .post(authServerUrl + "/realms/master/protocol/openid-connect/token")
-                .as(AccessTokenResponse.class).getToken();
+                .param("client_id", "admin-cli");
+
+        Response response = requestSpecification.when().post(authServerUrl + "/realms/master/protocol/openid-connect/token");
+
+        while (response.getStatusCode() != 200) {
+            // the Keycloak admin user isn't available yet, keep trying until it is to ensure we can obtain the token
+            // needed to set up the realms for the test
+            response = requestSpecification.when().post(authServerUrl + "/realms/master/protocol/openid-connect/token");
```
Comment (jamezp) [inline testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java:92]: I feel like we should add some kind of timeout here to be safe in cases where maybe the server never starts.

**Review by jamezp — APPROVED** [review]

File: testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java (hunk 1 above)
Comment (fjuma) [reply testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java:92]: Thanks, James! Will update this soon.

Comment (fjuma) [reply testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java:93]: @jamezp I've added a timeout now, thanks!

**Review by jamezp — APPROVED** [review]

**Comment by fjuma** [comment]
@jamezp I think we can merge this one now that it's run successfully multiple times.

**Comment by jamezp** [comment]
This doesn't have the two approvals, but I'm merging because it's re-enabling a test and only affects the test suite.

**Pull request merged**